altair/examples/airports_count.py

```diff
@@ -0,0 +1,47 @@
+"""
+Locations of US Airports
+========================
+This is a layered geographic visualization that shows the positions of US
+airports on a background of US states.
+"""
+# category: maps
+import altair as alt
+from vega_datasets import data
+
+airports = data.airports()
+airports_counts = airports[['state','latitude', 'longitude']]
```

Comment:
This line can be deleted, because the variable is redefined below.

Reply:
yes, thank you, I have overlooked it.